### ISSUE TITLE
[question] redirect_uri_path for multiple upstream services

### ISSUE THREAD

**Issue by illyaMs** [nokia/kong-oidc#118]
Hi there (and thanks for the great work).

I'm trying to protect several internal `k8s` services by exposing appropriate `kong` services with current plugin being activated. Google is used as an OIDC provider.

The problem I face is with setting the proper `redirect_uri_path` value in plugin settings. Since we're forced to add the whitelist of `Authorized redirect URIs` on google side, it seems like I have to add there all the kong `routes` paths for all the services I'm trying to protect. This doesn't look like a scale-able solution though, I would rather add any single trusted endpoint pointing to `kong`\`kong-oidc` and let the plugin or kong itself redirect further to a service that was initially requested.

The question is: is it possible via current plugin? Maybe chaining it with some other plugin/tool for certain kong services/routes would make the job done? Thanks in advance.

**Comment by sathishkumar-p** [comment]
Able to find solution ?

**Comment by illyaMs** [comment]
no, don't remember the details now, but I didn't proceed with that idea in the end
